universal-login-relayer/src/core/services/EmailService.ts

```diff
@@ -0,0 +1,87 @@
+import nodemailer, {SentMessageInfo} from 'nodemailer';
+import {MailOptions} from 'nodemailer/lib/json-transport';
+import Mail from 'nodemailer/lib/mailer';
+import {CannotSendEmail} from '../utils/errors';
+
+export class EmailService {
+  private transporter: Mail;
+  constructor(transporterEmail: string, transporterEmailPassword: string) {
+    this.transporter = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        user: transporterEmail,
+        pass: transporterEmailPassword,
+      },
+    });
+  }
+
+  async sendMail(mailOptions: MailOptions): Promise<SentMessageInfo> {
+    try {
+      return this.transporter.sendMail(mailOptions);
+    } catch (error) {
+      throw new CannotSendEmail(error.message);
+    }
```

Comment:
It won't work, because we need to wait for this error to be thrown

Reply:
Done